lib/diagnostic.rb

```diff
@@ -67,8 +84,9 @@ def initialize
 
 ##
 # your answers here
-Answer.housecat_noise = nil
-#
+Answer.housecat_noise = "I am a Cat, and I go meow"
```

Comment:
test wanted `Answer.housecat_noise = "I am a HouseCat, and I go 'meow'"` but i'll give it to you